### ISSUE TITLE
Build error because of reference to not existing file src/scanner.cc

### ISSUE THREAD

**Issue by jamilraichouni** [nickel-lang/tree-sitter-nickel#40]
Hi!

When I run `:Lazy update` I get an error:

```
Downloading tree-sitter-nickel...                                                                                                                                                                                                                                                          
Creating temporary directory                                                                                                                                                                                                                                                               
Extracting tree-sitter-nickel...                                                                                                                                                                                                                                                           
Compiling...                                                                                                                                                                                                                                                                               
cc1plus: fatal error: src/scanner.cc: No such file or directory                                                                                                                                                                                                                            
compilation terminated.                                                                                                                                                                                                                                                                    
Error during compilation                                                                                                                                                                                                                                                                   
Failed to execute the following command:                                                                                                                                                                                                                                                   
{                                                                                                                                                                                                                                                                                          
  cmd = "cc",                                                                                                                                                                                                                                                                              
  err = "Error during compilation",                                                                                                                                                                                                                                                        
  info = "Compiling...",                                                                                                                                                                                                                                                                   
  opts = {                                                                                                                                                                                                                                                                                 
    args = { "-o", "parser.so", "-I./src", "src/parser.c", "src/scanner.cc", "-Os", "-shared", "-lstdc++", "-fPIC" },                                                                                                                                                                      
    cwd = "/home/nörd/.local/share/nvim/tree-sitter-nickel"                                                                                                                                                                                                                                
  }                                                                                                                                                                                                                                                                                        
} 
```
There is a file `src/scanner.c` but not a `src/scanner.cc`:

<img width="550" alt="image" src="https://github.com/nickel-lang/tree-sitter-nickel/assets/45113557/e1208cfa-e4ab-45b6-a3a8-65c7fb5e099c">

**Comment by jamilraichouni** [comment]
Tried to push a branch with a potential fix but had an auth issue.
Maybe the following fixes this issue already:

<img width="927" alt="image" src="https://github.com/nickel-lang/tree-sitter-nickel/assets/45113557/9c21273b-d4b5-438b-8888-a4b133eb965c">

**Comment by vkleen** [comment]
Thanks a lot for reporting this! I'm confident that's fallout from #29. I've pushed your fix to my fork and made #41. Is there an easy way for you to verify that it solves the issue?

**Comment by jamilraichouni** [comment]
Hi, since this comes indirectly via tree-sitter and is not an Nvim plugin I have no clue how to test the code on a dedicated branch. What I can say is that renaming the file and re-running the command logged in the error message let the command (on the first screenshot here in this thread) run through. So I am quite confident that this will fix it.
